__tests__/integration/template-format.test.ts

```diff
@@ -109,6 +109,36 @@ describe("query using template format", () => {
     expect(response.data).toBe(true);
   });
 
+  it("succeeds with expressions nested within objects", async () => {
```

Comment:
i would like to see at a test for mulitple layers of nesting. Like:

const arg = {
  a: fql`1 + ${fql`2`}`
}
queryBuilder = fql`${arg}`

Reply:
i've added such a test